### ISSUE TITLE
The getter 'name' isn't defined for the class

### ISSUE THREAD

**Issue by ThangD** [Stacked-Org/stacked#479]
ERROR: ../../../../snap/flutter/common/flutter/.pub-cache/hosted/pub.dartlang.org/stacked_services-0.8.12/lib/src/route_observer.dart:15:18: Error: The getter 'name' isn't defined for the class 'GetDialogRoute<dynamic>'.

**Comment by liveaffiliates** [comment]
This just happened to me an hour ago as well. Tried downgrading but it didn't help.

**Comment by liveaffiliates** [comment]
I think it may be an update of the Get package which stacked services depends on

**Comment by carlomigueldy** [comment]
Getting the exception too.
```
/C:/development/flutter/.pub-cache/hosted/pub.dartlang.org/stacked_services-0.8.12/lib/src/route_observer.dart:15:18:
Error: The getter 'name' isn't defined for the class 'GetDialogRoute<dynamic>'.
 - 'GetDialogRoute' is from 'package:get/get_navigation/src/dialog/dialog_route.dart'
 ('/C:/development/flutter/.pub-cache/hosted/pub.dartlang.org/get-4.3.4/lib/get_navigation/src/dialog/dialog_route.dart') .
Try correcting the name to the name of an existing getter, or defining a getter or field named 'name'.
    return route.name;
                 ^^^^
/C:/development/flutter/.pub-cache/hosted/pub.dartlang.org/stacked_services-0.8.12/lib/src/route_observer.dart:19:18:
Error: The getter 'name' isn't defined for the class 'GetModalBottomSheetRoute<dynamic>'.
 - 'GetModalBottomSheetRoute' is from 'package:get/get_navigation/src/bottomsheet/bottomsheet.dart'
 ('/C:/development/flutter/.pub-cache/hosted/pub.dartlang.org/get-4.3.4/lib/get_navigation/src/bottomsheet/bottomsheet.da rt').
Try correcting the name to the name of an existing getter, or defining a getter or field named 'name'.
    return route.name;
                 ^^^^
/C:/development/flutter/.pub-cache/hosted/pub.dartlang.org/stacked_services-0.8.12/lib/src/route_observer.dart:61:9:     
Error: The setter 'reference' isn't defined for the class '_GetImpl'.
 - '_GetImpl' is from 'package:get/get_core/src/get_main.dart'
 ('/C:/development/flutter/.pub-cache/hosted/pub.dartlang.org/get-4.3.4/lib/get_core/src/get_main.dart').
Try correcting the name to the name of an existing setter, or defining a setter or field named 'reference'.
    Get.reference = newRoute.name;
        ^^^^^^^^^
/C:/development/flutter/.pub-cache/hosted/pub.dartlang.org/stacked_services-0.8.12/lib/src/route_observer.dart:84:9:     
Error: The setter 'reference' isn't defined for the class '_GetImpl'.
 - '_GetImpl' is from 'package:get/get_core/src/get_main.dart'
 ('/C:/development/flutter/.pub-cache/hosted/pub.dartlang.org/get-4.3.4/lib/get_core/src/get_main.dart').
Try correcting the name to the name of an existing setter, or defining a setter or field named 'reference'.
    Get.reference = newRoute.name;
        ^^^^^^^^^
/C:/development/flutter/.pub-cache/hosted/pub.dartlang.org/stacked_services-0.8.12/lib/src/route_observer.dart:108:9:    
Error: The setter 'reference' isn't defined for the class '_GetImpl'.
 - '_GetImpl' is from 'package:get/get_core/src/get_main.dart'
 ('/C:/development/flutter/.pub-cache/hosted/pub.dartlang.org/get-4.3.4/lib/get_core/src/get_main.dart').
Try correcting the name to the name of an existing setter, or defining a setter or field named 'reference'.
    Get.reference = newName;
```

**Comment by carlomigueldy** [comment]
Created a Pull Request to fix this issue

https://github.com/FilledStacks/stacked/pull/480

**Comment by FilledStacks** [comment]
Released in latest stacked services 0.8.13
